packages/hdwallet-core/src/wallet.ts

```diff
@@ -19,7 +19,8 @@ export interface GetPublicKey {
   showDisplay?: boolean,
   scriptType?: BTCInputScriptType,
   curve: string,
-  coin?: Coin
+  coin?: Coin,
+  symbol?: Symbol
```

Comment:
You should key this off of what the `coin` is, instead of adding another one.

Reply:
Symbol is already on the payload we are passing in which is why I added that as an optional key

Reply:
yeah, I see why you did that now... lemme think about it for a bit.